orchestration/dagster_orchestration/hca_manage/manage.py

```diff
@@ -182,18 +182,6 @@ def get_null_filerefs(self, target_table: str) -> set[str]:
 
         return self._hit_bigquery(query)
 
-    def _hit_bigquery(self, query: str) -> set[str]:
```

Comment:
❓ I assume the premise here is to keep the private methods at the bottom of the file?

Reply:
that's the idea, yeah. i'm considering adding a comment divider.

my other thought was just that the delineation between which methods are public and which are private is a bit unclear, but that's a topic for the refactor